app/controllers/admin/sessions_controller.rb

```diff
@@ -29,7 +29,7 @@ def edit; end
 
   def show
     @q = @session.missions.ransack(params[:q])
-    @missions = @q.result&.page(params[:page] || 1)&.per(20)
+    @missions = @q.result&.page(params[:page] || 1).per(20)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Lint/SafeNavigationChain: Do not chain ordinary method call after safe navigation operator.